cluster/saltbase/salt/etcd/init.sls

```diff
@@ -34,6 +26,28 @@ delete_etcd_initd:
   file.absent:
     - name: /etc/init.d/etcd
 
+#stop legacy etcd_service
+stop_etcd-service:
+  service.dead:
+    - name: etcd
+    - enable: None
+
+{% endif %}
+
+touch /var/log/etcd.log:
+  cmd.run:
+    - creates: /var/log/etcd.log
+
+/var/etcd:
```

Comment:
No.

Even though we run the default kubernetes etcd via a pod, we bind mount that directory in for logging, so we need to touch the files first in order to allow the mount to proceed.

See https://github.com/kubernetes/kubernetes/blob/master/cluster/saltbase/salt/etcd/etcd.manifest#L41